docs/reference/changelog.rst

```diff
@@ -33,6 +33,8 @@ Bug fixes:
 
 - The ``--recurse-submodules`` link in the ``source-submodules`` docstring
   no longer produces a linkcheck error.
+- ``CRAFT_DEFAULT_*`` environment variables are set when the default partition 
```

Comment:
Just to confirm: is there a released version of craft-parts that has this bug? Or did the bug + fix happen between releases?

Reply:
This bug is here since 2.16.0, when the alias mechanism was added.